crates/codegen/runtime/cargo/src/runtime/generated/kinds.rs

```diff
@@ -14,6 +14,7 @@ use napi_derive::napi;
     strum_macros::AsRefStr,
     strum_macros::Display,
     strum_macros::EnumString,
+    strum_macros::IntoStaticStr,
```

Comment:
I wonder if we need `IntoStaticStr` in addition to the existing `AsRefStr`. If the `static` lifetime is needed, is it possible to remove the latter?

Reply:
They are both needed I have discovered.

Reply:
I removed them, and replaced a single usage of `.as_ref()` in our tests with `.into()`, and it is passing correctly. I will submit it for review if you don't mind.